ApeSwapFinance/frontend
# [REFACTOR] - Add React-query to TitleCards

## ⛑️ **What was done:**
- Installed and configured React-query
- Added two custom hooks to fetch, cache and infer types to data
- Added `getStaticProps` to fetch the data on the server
- Removed redux reducers, actions and custom hooks associated with industry average

### 🧪 **How to test:**
- Visit `/liquidity-health`
- Title cards should be populated with data on the first load
- Check the network tab and filter for `/industry-stats`, each endpoint should have been called only once

### Before: ###

https://github.com/ApeSwapFinance/frontend/assets/47801291/f94e706d-aa31-4dde-8db7-8f0832222dc5

### After: ###

https://github.com/ApeSwapFinance/frontend/assets/47801291/919766bb-dc0f-4ad2-ba1b-d9792d4649e6
